Modix.Data/Models/Core/GuildUserEntity.cs

```diff
@@ -1,4 +1,5 @@
 ﻿using System;
+using System.Collections.Generic;
```

Comment:
Added by mistake? Not seeing any new code that uses this

Reply:
Yep - removed.